gbols/fast-food
# #160544191 validation added to ensure id is an integer.

**What does this PR?**
Ensure that the id parameter must be of type integer.

**Description of Task to be Completed?**
Redesign the UI.

**How should this be manually tested?**
 - pull from this branch  `bg-typeof-input-must-be-a-number-160544191` into a new branch 
 - run `npm install`
 - endpoint should look like this: `http://localhost:3000/api/v1/orders/1`.

**What are the relevant pivotal tracker stories?**
#160544191 Id passed as a parameter must be an integer.

**Postman Screenshots.**
![screenshot from 2018-09-17 09-26-36](https://user-images.githubusercontent.com/27724644/45613787-465d9e00-ba5f-11e8-8ca0-84d33cec9444.png)
